### DISCUSSION TITLE
how do I create new table with javascript

### DISCUSSION THREAD

**Discussion by dziubek89** [supabase/supabase#14466] [General]
Hello guys, 
I would like to allow users to create new projects on my page. When the user creates a new project it should create a new table with the name passed in the parameters in the supabase. Any idea how do I do this?

**Comment by kouwasi** [comment]
have you tried a trigger function? it's useful for that requirement i guess.

**Reply by dziubek89** [reply]
yes I tried but I got error: 
{
  code: '42501',
  details: null,
  hint: null,
  message: 'permission denied for schema public'
}
Any idea how to add permissions
